micro-rdk/src/common/ota.rs

```diff
@@ -64,7 +67,14 @@ use {bincode::Decode, futures_lite::AsyncWriteExt};
 const CONN_RETRY_SECS: u64 = 1;
 const NUM_RETRY_CONN: usize = 3;
 const DOWNLOAD_TIMEOUT_SECS: u64 = 30;
-const SIZEOF_APPDESC: usize = 256;
```

Comment:
looks like this is still being referenced in L. 481 (in the native case), probably meant to replace with `FIRMWARE_HEADER_SIZE`?